### DISCUSSION TITLE
umi约定式路由无法和antd-pro-layout一起使用

### DISCUSSION THREAD

**Discussion by fatalstar** [umijs/umi#6134] [Q&A] [answered]
操作步骤：配置中开启layout
1. 如果使用配置路由，可以写如下配置：这样可以让菜单显示当前路由
{
      exact: true, path: '/login',
      component: './login/index',
      menu: {
        name: 'overview',
        icon: 'testicon',
        flatMenu: false,
        hideInMenu: false,
        hideChildrenInMenu: false,
      },
    },
2. 如果使用约定式路由，并且使用扩展路由属性：会发现菜单并没有显示当前路由，而router里是添加进menu信息的
Login.menu ={
  name: 'overview',
  icon: 'testicon',
  flatMenu: false,
  hideInMenu: false,
  hideChildrenInMenu: false,
};

**Comment by fatalstar** [comment, marked as answer]
使用方法2时，需要把layout文件夹屏蔽掉。

**Reply by hideInEye** [reply]
登陆后重新设置权限路由显示。有解决过吗
